bin/runtests.php

```diff
@@ -0,0 +1,490 @@
+<?php
```

Comment:
Why do we need two different files for that? That's also something I've asked myself.